Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -324,13 +321,42 @@ extension ActivityStreamPanel {
             let touchPoint = longPressGestureRecognizer.locationInView(self.view)
             if let indexPath = tableView.indexPathForRowAtPoint(touchPoint) {
                 if Section(indexPath.section) == .Highlights {
-                    presentContextMenu(history[indexPath.row])
+                    presentContextMenuForHighlights(history[indexPath.row])
+                } else {
+                    let topSiteCell = self.tableView.cellForRowAtIndexPath(indexPath) as! ASHorizontalScrollCell
+                    let touchPointWithinTopSiteCell = longPressGestureRecognizer.locationInView(topSiteCell.collectionView)
```

Comment:
I had trouble getting the collection view underlying the top sites section. This was essential for the "Remove" action. So, I made the `ASHorizontalScrollCell.collectionView` property private.